test/collection-facets/facets-template.test.ts

```diff
@@ -46,158 +46,120 @@ describe('Render facets', () => {
         .renderOn=${'modal'}
       ></facets-template>`
     );
-    await el.updateComplete;
 
     expect(el.shadowRoot?.querySelector('.facets-on-modal')).to.exist;
   });
 
-  it('find facet-title and facet-count for particular facet group', async () => {
```

Comment:
Some old `facet-template` tests have been migrated into equivalent ones in `facet-row.test.ts`.